galaxy.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   community.crypto: '>=2.2.3'
   community.general: '>=4.5.0'
   kubernetes.core: '>=2.3.2'
-  vexxhost.containers: '>=1.3.0'
+  vexxhost.containers: '>=1.5.0'
```

Comment:
I think we should release 1.6.0 since it has a few improvements to support newer Python.